Fakebook.Posts/Fakebook.Posts.Domain/Models/Comment.cs

```diff
@@ -17,9 +17,17 @@ public Comment(string userEmail, string content)
         {
             if (string.IsNullOrWhiteSpace(userEmail)) throw new ArgumentException("User email is required.", nameof(userEmail));
             if (string.IsNullOrWhiteSpace(content)) throw new ArgumentException("Post content is required.", nameof(content));
+            try
+            {
+                userEmail = new MailAddress(userEmail).Address;
+            }
+            catch (FormatException e)
+            {
+                throw new ArgumentException("Enter a correct email format.", nameof(userEmail), e);
+            }
             UserEmail = userEmail;
             Content = content;
             Likes = new HashSet<string>();
         }
     }
-}
+}
```

Comment:
fix the code smell, don't remove the newline at end of file. same for the post.cs file.

Reply:
I added a new line at the end of this file.